core/src/main/scala/kafka/utils/Json.scala

```diff
@@ -78,4 +79,11 @@ object Json {
    */
   def encodeAsBytes(obj: Any): Array[Byte] = encode(obj).getBytes(StandardCharsets.UTF_8)
 
+  def encodeToJsonString(obj: Map[String,Any]): String = {
+     mapper.writeValueAsString(obj.asJava)
+  }
+
+  def encodeToJsonBytes(obj: Map[String,Any]): Array[Byte] = {
```

Comment:
This should be `encodeAsBytes` and it should take `Any` as the type. We can maybe rename the existing `encodeAsBytes` to `legacyEncodeAsBytes`.

Reply:
This change is done.